LibreNMS/OS/Vrp.php

```diff
@@ -133,7 +133,7 @@ public function pollOS(DataStorageInterface $datastore): void
                 'hwWlanRadioChUtilizationRate',
                 'hwWlanRadioChInterferenceRate',
                 'hwWlanRadioActualEIRP',
-                'hwWlanRadioFreqType',
+                'hwWlanRadioType',
```

Comment:
For consistancy with other vendors, changing to RadioType. Bitwise masks used to decode the OID and concatenate the string type.